### PR TITLE
Data replication configuration for EndPoints

### PR DESCRIPTION
**Pull request by tuGithub**

### CODE REVIEW

**Comment by tuGithub** [comment]
@chavdar , please review when you have time

**Comment by chavdar** [comment]
> Execution failed for task ':gobblin-data-management:findbugsMain'.
>> FindBugs rule violations were found. See the report at: file:///home/travis/build/linkedin/gobblin/build/gobblin-data-management/reports/findbugs/main.html

**Comment by tuGithub** [comment]
fixed the FindBug warning

File: gobblin-data-management/src/main/java/gobblin/data/management/copy/replication/ReplicaHadoopFsEndPoint.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  }
+
+  @Override
+  public Watermark getWatermark() {
+    LongWatermark result = new LongWatermark(-1);
+    try {
+      Path metaData = new Path(rc.getPath(), WATERMARK_FILE);
+      FileSystem fs = FileSystem.get(rc.getFsURI(), new Configuration());
+      if (fs.exists(metaData)) {
+        FSDataInputStream fin = fs.open(metaData);
```
Comment (chavdar) [inline gobblin-data-management/src/main/java/gobblin/data/management/copy/replication/ReplicaHadoopFsEndPoint.java]: Please use try-resources block to make sure the inputstream is closed.

Comment (tuGithub) [inline gobblin-data-management/src/main/java/gobblin/data/management/copy/replication/ReplicaHadoopFsEndPoint.java]: fixed

**Pull request merged**